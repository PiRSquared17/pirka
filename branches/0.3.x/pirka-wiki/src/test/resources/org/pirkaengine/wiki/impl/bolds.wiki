This *paragraph* contains *strong* text.
