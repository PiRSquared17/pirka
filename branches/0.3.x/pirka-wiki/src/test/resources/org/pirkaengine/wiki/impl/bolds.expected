<p>This <strong>paragraph</strong> contains <strong>strong</strong> text.</p>
